lms/djangoapps/grades/tasks.py

```
@@ -187,6 +187,7 @@ def _recalculate_subsection_grade(self, **kwargs):
             scored_block_usage_key,
             kwargs['only_if_higher'],
             kwargs['user_id'],
+            kwargs['score_deleted'],
```

Note: this will cause errors during blue-green deployment since old-versions won't be sending this parameter.  You'll want to create a `recalculate_subsection_grade_v4` to address the backward-compatibility issue.

---

Could I also just change this to `kwargs.get('score_deleted', False)`?

---

Looking at this further, I'm not sure I understand why I need to change anything here; the kwargs being passed to the workers haven't changed.

---

oh.  If all callers to recalculate_subsection_grade already previously sent a value for score_deleted, then, yes, you're right no change is needed here.

---

That's already happening in handlers.py, so I think it's redundant. There's no chance the caller won't provide the kwarg.

---

Although this is a mute point at this point, I tend to steer away from having overly defensive code.  If we did change this to a `get`, we would mask potential errors - that would make downstream errors even harder to debug.